plugins/spu2-x/src/Config.h

```diff
@@ -99,5 +99,4 @@ extern void ApplySettings(soundtouch::SoundTouch &sndtouch);
 
 extern void ReadSettings();
 extern void WriteSettings();
-extern void configure();
-extern void AboutBox();
+extern void configure();
```

Comment:
I'd prefer keeping the newline (my Linux text editor (vim) will re-introduce one if I modify the file).